awscrt/s3.py

```diff
@@ -68,8 +68,12 @@ class S3Client(NativeResource):
             If this is :attr:`S3RequestTlsMode.DISABLED`:
                 No TLS options will be used, regardless of `tls_connection_options` value.
 
-        credential_provider (Optional[AwsCredentialsProvider]): Credentials providers source the
-            :class:`~awscrt.auth.AwsCredentials` needed to sign an authenticated AWS request.
+        signing_config (Optional[AwsSigningConfig]):
+             Configuration for signing of the client. Use `create_default_s3_signing_config` to create the default config.
```

Comment:
trivial: do a link
```suggestion
             Configuration for signing of the client. Use :func:`create_default_s3_signing_config()` to create the default config.
```

you can check how the docs look locally by running `./scripts/make-docs.py`